ground/src/main/java/com/google/android/ground/model/mutation/LocationOfInterestMutation.kt

```diff
@@ -31,7 +32,7 @@ data class LocationOfInterestMutation(
   override val retryCount: Long = 0,
   override val lastError: String = "",
   val jobId: String = "",
-  val geometry: Geometry? = null,
+  val geometry: GeometryWrapper? = null,
```

Comment:
Maybe this should be not null?

Reply:
According to the LocationOfInterestMutationEntity this can be null if the geometry was unchanged in the mutation.

```
/** Non-null if the LOI's geometry was updated, null if unchanged. */
```